### PR TITLE
fixed some odd behavior when changing properties of cluster collections

### PR DESCRIPTION
**Pull request by jsteemann**
Docs PR for https://github.com/arangodb/arangodb/pull/12056

~TODO: Move changes over to 3.9 folder once it exists.~

### CODE REVIEW

File: 3.8/release-notes-api-changes38.md [hunk 1]
```diff
@@ -11,6 +11,49 @@ integrations for ArangoDB 3.8.
...
+In previous versions, the `replicationFactor` and `writeConcern` properties
+returned for a collection that was created with the `distributeShardsLike`
+option were sealed and did not reflect any changes done to these properties 
+in the prototype collection afterwards.
+Now the properties returned are always up-to-date and reflect the _current_
+status of these properties in the prototype collection.
+
+#### Other changes
+
+Trying to change the "writeConcern" value of an existing collection that uses
```
Comment (Simran-B) [inline 3.8/release-notes-api-changes38.md:49]:
```suggestion
Trying to change the `writeConcern` value of an existing collection that uses 
```

File: 3.8/release-notes-upgrading-changes38.md [hunk 2]
```diff
@@ -55,6 +57,29 @@ Client applications and tests that rely on the behavior that setting any of
...
+In previous versions, the `replicationFactor` and `writeConcern` properties
+returned for a collection that was created with the `distributeShardsLike`
+option were sealed and did not reflect any changes done to these properties 
+in the prototype collection afterwards.
+Now the properties returned are always up-to-date and reflect the _current_
+status of these properties in the prototype collection.
+
+### Other changes
+
+Trying to change the "writeConcern" value of an existing collection that uses
```
Comment (Simran-B) [inline 3.8/release-notes-upgrading-changes38.md:75]:
```suggestion
Trying to change the `writeConcern` value of an existing collection that uses 
```

File: 3.8/release-notes-upgrading-changes38.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Foxx apps are used.
 In ArangoDB 3.8 the default value for the option is `false`, meaning a server will
 complete the boot sequence faster, and the Foxx services will be synchronized in 
 a background operation. Until that operation has completed, any requests to a 
-Foxx app may be responded to with an HTTP 500 error and message 
+Foxx app may be responded to with an HTTP 503 error and message
```
Comment (Simran-B) [inline 3.8/release-notes-upgrading-changes38.md:31]: Was this changed for 3.8?

**Comment by ansoboleva** [comment]
closed as obsolete

**Comment by Simran-B** [comment]
@ansoboleva The upstream PR is still open, so what's the policy here?

**Comment by ansoboleva** [comment]
@Simran-B as no development work is planned for it anytime soon (discussed it with @jsteeman) and as part of cleaning out the PR list, i decided to close this PR. The general policy is not to have doc items in limbo - sitting in progress for months on end. This is required to provide a clear overview of the current tasks for resource evaluation/allocation purposes in the doc team.

**Pull request closed without merging**